include/rocksdb/statistics.h

```diff
@@ -488,6 +488,8 @@ struct HistogramData {
 // Usage:
 //   options.statistics->set_stats_level(StatsLevel::kExceptTimeForMutex);
 enum StatsLevel : uint8_t {
+  // Disable tickers
+  kExceptTickers,
```

Comment:
Should it be `kDisable`? `kExceptTickers` sounds like there are metrics left without tickers.

Reply:
I see. Currently there are no other metrics if we exclude `kExceptTickers` and above.
Though highly unlikely, but will it be possible that one day in the future we are going to add something between `disable` and `except tickers`?
How about the following
```
enum StatsLevel : uint8_t {
  kDisable,
  kExceptTickers = kDisable,
  ...
};
```